src/components/shared/Tag.js

```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+import { Link } from "@reach/router";
```

Comment:
I've seen this being imported from here and also imported from "gatsby". Do we know what the difference is for that and whichever one we decide on, we should make consistent

Reply:
I can update all to use Link from gatsby.

this is stackoverflow answer
```Gatsby Link is a required wrapper for the react-router-dom Link component. The two work the same but Gatsby's version adds prefetching as well as prevents route transitions until the next page's resources are finished loading.

Things will break in weird ways if you use React Route <Link>.```

Reply:
cool, thanks for looking into that